mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -1931,14 +2013,14 @@ void CMomentumPlayer::ApplyPushFromDamage(const CTakeDamageInfo &info, Vector &v
         return;
 
     CBaseEntity *pAttacker = info.GetAttacker();
+    CBaseEntity *pInflictor = info.GetInflictor();
 
     if (!info.GetInflictor() || GetMoveType() != MOVETYPE_WALK || pAttacker->IsSolidFlagSet(FSOLID_TRIGGER))
         return;
 
-    // Apply different force scale when on ground
-    float flScale = 1.0f;
+    float flScale = 0.0f;
```

Comment:
I'd say it would be better to have a scale of 0 instead of 1 if some unknown entity tries to give knockback without explicitly coded knockback values for it to use.